oximeter/db/src/lib.rs

```diff
@@ -35,11 +35,12 @@ pub mod model;
 #[cfg(any(feature = "oxql", test))]
 pub mod oxql;
 pub mod query;
-#[cfg(any(feature = "oxql", feature = "sql", test))]
+#[cfg(any(feature = "oxql", feature = "sql", feature = "oxdb", test))]
```

Comment:
`oxdb` exposes the shells mod for the other commands, e.g. wipe, query, that are not `oxql`/`sql`.